components/input/input.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sinister-UI | Input</title>
+    <link rel="icon" type="image/x-icon" href="../../assets/Sinister-UI.svg">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="../../components/main.css">
+    <!-- <link rel="stylesheet" href="../../components/typography/typography.css"> -->
+    <link rel="stylesheet" href="../../docs.css">
+    <link rel="stylesheet" href="../../landing.css">
+</head>
+<body>
+    <header class="navbar" >
+        <p class="navbar-brand">
+            <a href="../../index.html"  >Sinister UI</a>
+            <small class="subtitle2 small-text" >V1.0</small>
```

Comment:
what does subtitle2 mean?
I suppose it should be subtitle-lg for large subtitle